### ISSUE TITLE
Wrong bond endpoint when connecting to wedge bond in 2D image

### ISSUE THREAD

**Issue by stgeo** [rdkit/rdkit#5963]
**Describe the bug**
When drawing a 2D depiction of a particular molecule with a wedge bond, the bond that should connect to the wide end instead connects in the middle of the bond.

**To Reproduce**
```
from rdkit.Chem.Draw import rdMolDraw2D
from rdkit.Chem import AllChem

molblock = """S-MeOPh-benzyl-sulfoxide
  TEST    09202213353D

  0  0  0     0               999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 31 32 0 0 0
M  V30 BEGIN ATOM
M  V30 1 H 0.196641 -3.231362 -1.275853 0
M  V30 2 C 0.973113 -2.604889 -0.845277 0
M  V30 3 C 1.847067 -3.128677 0.106455 0
M  V30 4 H 1.754109 -4.164755 0.420270 0
M  V30 5 C 2.845742 -2.321288 0.647379 0
M  V30 6 H 3.533853 -2.726819 1.384112 0
M  V30 7 C 2.968949 -0.994892 0.242774 0
M  V30 8 H 3.752968 -0.368401 0.659016 0
M  V30 9 C 2.092088 -0.459547 -0.707867 0
M  V30 10 C 2.182272 0.972547 -1.088004 0
M  V30 11 C 1.091993 -1.278564 -1.246251 0
M  V30 12 H 0.405267 -0.868972 -1.983370 0
M  V30 13 H -0.047664 0.197905 1.871936 0
M  V30 14 C -0.725661 0.437131 1.057415 0
M  V30 15 C -0.331049 1.333587 0.076647 0
M  V30 16 C -1.186023 1.669476 -0.974122 0
M  V30 17 H -0.877638 2.383935 -1.734156 0
M  V30 18 C -2.435271 1.079305 -1.050264 0
M  V30 19 H -3.117739 1.320322 -1.859764 0
M  V30 20 C -2.840633 0.163867 -0.066533 0
M  V30 21 O -4.085639 -0.352836 -0.228955 0
M  V30 22 C -1.985732 -0.153642 0.994275 0
M  V30 23 H -2.281669 -0.852730 1.767733 0
M  V30 24 H 3.207665 1.356530 -1.081448 0
M  V30 25 H 1.708344 1.201171 -2.048311 0
M  V30 26 S 1.305074 2.096004 0.164211 0
M  V30 27 O 1.881505 1.740943 1.536027 0
M  V30 28 C -4.554597 -1.290776 0.751308 0
M  V30 29 H -5.556241 -1.576585 0.428703 0
M  V30 30 H -3.910361 -2.177249 0.785950 0
M  V30 31 H -4.604059 -0.827826 1.744021 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 1 1 1 2
M  V30 2 4 2 3
M  V30 3 4 2 11
M  V30 4 1 3 4
M  V30 5 4 3 5
M  V30 6 1 5 6
M  V30 7 4 5 7
M  V30 8 1 7 8
M  V30 9 4 7 9
M  V30 10 1 9 10
M  V30 11 4 9 11
M  V30 12 1 10 24
M  V30 13 1 10 25
M  V30 14 1 10 26
M  V30 15 1 11 12
M  V30 16 1 13 14
M  V30 17 4 14 15
M  V30 18 4 14 22
M  V30 19 4 15 16
M  V30 20 1 15 26
M  V30 21 1 16 17
M  V30 22 4 16 18
M  V30 23 1 18 19
M  V30 24 4 18 20
M  V30 25 1 20 21
M  V30 26 4 20 22
M  V30 27 1 21 28
M  V30 28 1 22 23
M  V30 29 2 26 27
M  V30 30 1 28 29
M  V30 31 1 28 30
M  V30 32 1 28 31
M  V30 END BOND
M  V30 END CTAB
M  END"""

mol = AllChem.MolFromMolBlock(molblock)

mol.Compute2DCoords()

d2d = rdMolDraw2D.MolDraw2DSVG(300,300)
d2d.DrawMolecule(mol)
d2d.FinishDrawing()

with open('drawing.svg', 'w') as f:
    f.write(d2d.GetDrawingText())
```

**Expected behavior / Screenshots**

The code above produces this in versions 2022.9.1 and 2022.9.3:
![drawing-2022 9 1](https://user-images.githubusercontent.com/13415193/211846189-ba0d3c80-c85d-4449-bef8-7c51affb212a.svg)
While it correctly produced this in version 2022.3.5:
![drawing-2022 3 5](https://user-images.githubusercontent.com/13415193/211846348-99d190c7-fec6-452f-ae7c-3f7fd9b38715.svg)

**Configuration (please complete the following information):**
 - RDKit version: 2022.9.3
 - OS: Ubuntu 22.04
 - Python version: 3.10
 - Are you using conda? No
 - If you are not using conda: how did you install the RDKit? `pip install rdkit`

**Comment by DavidACosgrove** [comment]
I can reproduce this in the current branch.  I'll take a look at it over the weekend.
I wonder if it's linked to the unusual stereochemistry, although as an AstraZeneca pensioner I feel I ought to be up to speed with this particular mode of chirality.

**Comment by greglandrum** [comment]
Closed in #5978
